jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/complex/bars/SystemBarsTests.java

```diff
@@ -44,6 +44,7 @@ public void coloredSystemBarTests(int barNum, String barText, String barColor) {
         coloredSystemBars.get(barNum).signalIcon().is().displayed();
         coloredSystemBars.get(barNum).batteryIcon().is().displayed();
         coloredSystemBars.get(barNum).has().color(barColor);
+        coloredSystemBars.get(barNum).timeSpan().has().text("12:30");
     }
 
```

Comment:
Please, add an example (additional test) of icon list usage from icons() and systemBar() usage

Reply:
Added additional tests for test IconBar class and to test icons() and systemBar() usages